src/cart/crt_init.c

```diff
@@ -82,7 +82,9 @@ crt_lib_init(void)
 	crt_gdata.cg_inited = 0;
 	crt_gdata.cg_primary_prov = CRT_PROV_OFI_TCP_RXM;
 
-	d_srand(d_timeus_secdiff(0) + getpid());
+	rc = d_gettime(&now);
+	D_ASSERTF(rc == 0, "d_gettime: " DF_RC "\n", DP_RC(rc));
+	d_srand(now.tv_sec * 1000 * 1000 * 1000 + now.tv_nsec + getpid());
```

Comment:
minor: i feel now.tv_nsec + getpid() is sufficient for seed. wont now.tv_sec * 1000 * 1000 * 1000 cause overflows? coverity might complain if so:)